### PR TITLE
Run and walk animation jitter correction

### PR DESCRIPTION
**Pull request by PalauiDCL**
When the speed of the character fluctuated fast between being above of the running speed and below it, the character animation glitched trying to crossfade between both animations.

Even though this PR should solve the issue by adding some animation time before changing again between running and walking, I propose adding a different threshold for the animations depending if we are already running or walking.

### CODE REVIEW

**Review by Kinerius — APPROVED** [review]
Looks good, left a comment :D

**Comment by menduz** [comment]
Testing with the kernel branch for the avatar simulation I can see that the avatars do moon-walking in any direction https://www.youtube.com/watch?v=Q2F_SUUHG6s&feature=youtu.be

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs [hunk 1]
```diff
@@ -287,12 +288,15 @@ void State_Ground(BlackBoard bb)
...
-
-        if (movementSpeed > runMinSpeed)
+        if (movementSpeed >= WALK_MAX_SPEED)
         {
             CrossFadeTo(AvatarAnimation.RUN, runAnimationName, RUN_TRANSITION_TIME);
         }
-        else if (movementSpeed > walkMinSpeed)
+        else if (movementSpeed >= RUN_MIN_SPEED && movementSpeed < WALK_MAX_SPEED) 
+        { 
+            // Keep current animation, leave empty
```
[the remark this answers is not in the document]
Comment (PalauiDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs:297]: There is a gap between walk max and run min, so while the avatar is between 4 - 6 speed can run or walk, keeping the current animation.

**Review by kwssbocian — APPROVED** [review]
Fix appears to be implemented correctly. No further issues were observed when testing both running and walking animations. Issue for characters moonwalking also no longer occurs after dev branch was merged in.

**Pull request merged**